moabb/datasets/bids_interface.py

```diff
@@ -352,7 +355,7 @@ def _write_file(self, bids_path, raw):
             format="EDF",
             allow_preload=True,
             montage=raw.get_montage(),
-            overwrite=False,
+            overwrite=True,
```

Comment:
super important when we are processing each subject in parallel @PierreGtch

Reply:
I think this is not a good idea, the cache system is not made for parallel processing at the moment. We should make it compatible with parallel processing in a separate issue